client/app.jsx

```diff
@@ -1,8 +1,106 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import parseRoute from './lib/parse-route';
 import Home from './pages/home';
+import Results from './pages/results';
+import NavBar from './components/nav-bar';
+import Banner from './components/banner';
+import Footer from './components/footer';
+import SearchModal from './components/search-modal';
+import SneakerCarousel from './pages/sneakerCarousel';
 
-export default class App extends React.Component {
-  render() {
-    return <Home />;
+export default function App() {
+  const [isVisible, setIsVisible] = useState(false);
+  const [searchTerm, setSearchTerm] = useState('');
+  const [searchResult, setSearchResult] = useState('');
```

Comment:
Since this holds an array, you should be consistent with your types and set this either to `undefined` or to `[]`.